src/sentry.configuration.ts

```diff
@@ -63,8 +60,11 @@ function configureLoggingSentry(
 
     beforeSend: (event, hint) => {
       const error = hint.originalException;
-      if (error instanceof UnauthorizedException) {
-        console.log(error);
+      if (
```

Comment:
we generally hardly have any logs in the container itself. We could "abuse" this function to log all errors to console as a side effect ...

Reply:
I'd rather add specific logs then, otherwise we'll just spam up our hard disk with (mostly) useless information.